src/translations/en.json

```diff
@@ -1611,9 +1611,9 @@
                 "elevation_meters": "meters",
                 "time_zone": "Time Zone",
                 "unit_system": "Unit System",
-                "unit_system_imperial": "Imperial",
```

Comment:
We might not want to delete the Imperial keys if they are planned to be added back.

Reply:
This is better discussed on the core PR.
https://github.com/home-assistant/core/pull/80253

If you look at this comment https://github.com/home-assistant/core/pull/80253#discussion_r994698166 if we bring back imperial it will be under a new name, maybe british_imperial

We are also considering SI but that would be a distinct system with Kelvin as base temperature unit... so we are keeping metric.

Reply:
> If you look at this comment [home-assistant/core#80253 (comment)](https://github.com/home-assistant/core/pull/80253#discussion_r994698166) if we bring back imperial it will be under a new name, maybe british_imperial

I think that's a mistake.  The dominant references use Imperial.  I've been an engineer for a long time and never once have I heard of the "British Imperial" system.

> We are also considering SI but that would be a distinct system with Kelvin as base temperature unit... so we are keeping metric.

Again, I don't understand the logic there.  Metric and SI are not distinct systems - one evolved into the other and has been internationally standardized.

IMO, you guys are putting way too much thought into unit systems instead of just the default units users expect for different types of quantities.

Reply:
Agreed. At this stage I am focused on fixing the "not actually imperial wrongly named  US system". That's the one creating confusion.
After that is done we can look again at the other systems (be it SI or imperial).
Baby steps...